website/docs/playground.mdx

```diff
@@ -21,4 +21,6 @@ import {PlaygroundCardsRow} from '@site/src/components/Playground';
 
 For convenience, we'll remember your choice next time you visit [docusaurus.new](https://docusaurus.new).
 
+If you prefer a TypeScript playground, visit [docusaurus.new/ts](https://docusaurus.new/ts) instead.
```

Comment:
Unfortunately, this link doesn't work, there's no configured serverless function to handle it.

Also, how would we automatically choose between Codesandbox and Stackblitz without biasing toward one in particular??

Reply:
Can't you refer to and reuse the cookie used by the URL without `ts` suffix?
I can't believe the existence of people using CodeSandBox for JS and StackBlitz for TS or vice versa.

Reply:
The cookie does not exist on the first visit. 

When you visit `/` it simply redirects to the playground page, not a specific sandbox. I could make `/ts` redirect to the playground page as well on the first visit if you want. Not sure it's worth it.